atomic_reactor/utils/operator.py

```diff
@@ -314,6 +314,11 @@ def __init__(self, path, data, pullspec_heuristic=default_pullspec_heuristic):
         self.data = data
         self._pullspec_heuristic = pullspec_heuristic
 
+    @property
+    def md5sum(self):
```

Comment:
If any of the concern about verifying that the CSVs match involves potential security implications from e.g. spoofing CSVs, you will definitely want to use something other than MD5 - it is no longer considered to be a "crypto" hash (it's not really any better than e.g. XOR).

Reply:
@ben-alkov Thanks for this info. Fixed.